modules/dxp/apps/analytics/analytics-reports-layout-display-page-api/src/main/java/com/liferay/analytics/reports/layout/display/page/info/item/LayoutDisplayPageObjectProviderAnalyticsReportsInfoItem.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.analytics.reports.layout.display.page.info.item;
+
+import com.liferay.analytics.reports.info.item.AnalyticsReportsInfoItem;
+import com.liferay.layout.display.page.LayoutDisplayPageObjectProvider;
+
+/**
+ * @author Cristina González
+ */
+public interface LayoutDisplayPageObjectProviderAnalyticsReportsInfoItem
+	extends AnalyticsReportsInfoItem<LayoutDisplayPageObjectProvider> {
```

Comment:
`LayoutDisplayPageObjectProvider`  should be parametrized, right?

Reply:
We can't :(, the tracker won't recognize a wildcard parametrized class, and this object won't be registered.